src/pages/index.js

```diff
@@ -226,12 +226,13 @@ const IndexPage = () => (
           </SprkStackItem>
 
           <SprkStackItem>
-            <SprkButton
+            <SprkLink
```

Comment:
This can stay a SprkButton. We have a prop here for element="a" for this specific use case.

Reply:
Thanks Amber! I made this change so that the Learn More link would be included in Link Navigation. [When rendering SprkButton as a link](https://react.sparkdesignsystem.com/?path=/story/components-button--as-a-link-element), the output is still tagged as semantically a button with `role="button"`. This causes it to not be included in Link Navigation. I don't think `SprkButton` works for our use case.

Reply:
Making a bug issue for this one as we chatted about.